setup.py

```diff
@@ -0,0 +1,32 @@
+import subprocess
+import argparse
+
+parser = argparse.ArgumentParser()
+parser.add_argument("--skip-ansible-install",
+                        action = "store_false",
+                        help = "allows skipping of ansible install")
+parser.add_argument("-c", "--currency-type",
+                        required = True,
+                        help="bitcoin, ethereum, litecoin")
+parser.add_argument("--smtp",
+                        action = "store_true",
+                        help = "if you have the smpt server, port, username, and password you can set up alerting")
+parser.add_argument("-d", "--dry-run",
+                       action = "store_true",
+                       help = "dry-run ansible playbook")
+args = parser.parse_args()
+
+#Install ansible. Allows the running of the ansible script
+if (args.skip_ansible_install):
+  subprocess.call("apt-get update", shell=True)
+  subprocess.call("apt-get upgrade -y", shell=True)
+  subprocess.call("apt-get install ansible -y", shell=True)
+
```

Comment:
dry run has some other dependency that I forget. Do not forget this otherwise dry-run will not work.

Reply:
Either way if its the case I missed something I'll open an issue against it